scripts/check-api-compatibility.sh

```diff
@@ -55,17 +55,23 @@ if ! ${SKIP_DOWNLOAD:-false}; then
     existing_names=$(echo "$jsii_package_dirs" | xargs -n1 -P4 -I {} bash -c 'dirs_to_existing_names "$@"' _ {})
     echo " Done." >&2
 
+    echo "Determining baseline version..." >&2
     version=$(node -p 'require("./scripts/resolve-version.js").version')
-    echo "Current version is $version."
+    echo "  Current version is $version." >&2
 
     if ! package_exists_on_npm aws-cdk $version; then
+      echo "  Version $version does not exist in npm. Falling back to env vars." >&2
       # occurs within a release PR where the version is bumped but is not yet published to npm.
-      if [ -z ${NPM_DISTTAG:-} ]; then
-        echo "env variable NPM_DISTTAG is not set. Failing."
-        exit 1
+      major_version=$(echo $version | cut -d '.' -f 1)
+      if [ "$major_version" == "1" ]; then
+        echo "  Setting version to $NPM_DISTTAG_V1" >&2
+        version=$NPM_DISTTAG_V1
+      fi
+
+      if [ "$major_version" == "2" ]; then
```

Comment:
Can you combine with the above (`elif`) and add an else, so when we release v3 (or screw up the version somehow) we get an explicit error?

Reply:
Done!